helm-charts/0.26.1/values.yaml

```diff
@@ -166,4 +166,6 @@ global:
     logLevel: DEBUG
     # -- The SPIRE Server port. This is the port where the SPIRE Server will
     # listen for incoming connections.
-    serverPort: 8081
+    serverPort: 443
+
+    bundleEndpointEnabled: false
```

Comment:
It looks like we don’t need a bundle endpoint unless we want to federate stuff.

Disabling it by default for added security.